packages/framework/src/Resources/config/services.yml

```diff
@@ -98,24 +98,20 @@ services:
             - '@annotation_reader'
 
     shopsys.doctrine.cache_driver.query_cache:
-        class: Doctrine\Common\Cache\ChainCache
+        class: Doctrine\Common\Cache\RedisCache
```

Comment:
does this mean that we removed chainCache so we would not use ArrayCache before Redis cache for reading?
is this PR really about performance improvement ? 
please contact also @pk16011990 , he got some pros for removing the chain cache and also for not removing the chain cache.

Reply:
We checked that on our internal project, when we increased performance by use Redis for this. And yes, we should test performance with this changes